cron/alerts/main.go

```diff
@@ -194,15 +194,17 @@ func getCFRecords(config AlertsConfig) (map[string][]string, error) {
 		zoneID, err := api.ZoneIDByName(zoneName)
 		if err != nil {
 			err = fmt.Errorf("error getting Cloudflare zone %s ... %v ", zoneName, err.Error())
-			return results, err
+			sendErrorEmails(config, err)
```

Comment:
An alternative way, but I am not sure of how it is formatted, would be to use `errors.Join()`, then could send only one email. Something like:

```
var errs error
for ... {
    zoneID, err := ...
    if err != nil {
        errs = errors.Join(errs, err)
        continue
    }

    .
    .
    .
}

return results, errs
```

Then can keep other code how was like. Note: this is only for go 1.20+. This may play better with the test

Reply:
Good to know there is a `Join` option. Since this isn't "production" code, I'm not concerned about returning an error. If your unmentioned "qualms" are significant, maybe you can elaborate.